### ISSUE TITLE
Loss Function Error for L2 YoloV3

### ISSUE THREAD

**Issue by LenaNans** [NeuralSec/Daedalus-attack#4]
Hello,
I am attempting to run the l2_yolov3 section of code and coming across the error:
"AbortedError (see above for traceback): Operation received an exception:Status: 3, message: could not create a dilated convolution forward descriptor, in file tensorflow/core/kernels/mkl_conv_ops.cc:1111".
I have traced this error to some of the loss function initializations/calculations once X_adv, distortions = attacker.attack(X_test) is executed. It looks like the loss values are not being stored properly in a tensor. This issue might be tied with the newest version of TF& Keras, but any help regarding this issue would be greatly appreciated.
Thank you for your time.
-Lena

**Comment by xubo0321** [comment]
> Hello,
> I am attempting to run the l2_yolov3 section of code and coming across the error:
> "AbortedError (see above for traceback): Operation received an exception:Status: 3, message: could not create a dilated convolution forward descriptor, in file tensorflow/core/kernels/mkl_conv_ops.cc:1111".
> I have traced this error to some of the loss function initializations/calculations once X_adv, distortions = attacker.attack(X_test) is executed. It looks like the loss values are not being stored properly in a tensor. This issue might be tied with the newest version of TF& Keras, but any help regarding this issue would be greatly appreciated.
> Thank you for your time.
> -Lena

@LenaNans    I have succeed with tensorflow 1.12 and the related keras and python2.7.
my conputer is 1050Ti, and it seemed very slowly when I put ten images in the data file.
